src/status_im/ui/screens/wallet/account/views.cljs

```diff
@@ -37,7 +38,7 @@
   [react/touchable-highlight {:on-press handler :style {:flex 1}}
    [react/view {:flex 1 :align-items :center :justify-content :center}
     [react/view {:flex-direction :row :align-items :center}
-     [icons/icon icon {:color color}]
+     [vector-icons/icon icon {:color color}]
```

Comment:
why renamed ?

Reply:
Oh, we shuffled things around, and imported and remamed etc, we'll change it back ;)

Reply:
Reverted